src/Assets/TestProjects/NetCoreCsharpAppReferenceCppCliLib/NETCoreCppCliTest/Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+  <PropertyGroup>
+    <ResolveNuGetPackages>false</ResolveNuGetPackages>
+    <EnableCppNETCoreSupport>true</EnableCppNETCoreSupport>
+    <EnableManagedIncrementalBuild>false</EnableManagedIncrementalBuild>
+
+    <!-- Workaround: These should be changed in Microsoft.CppCommon.targets to the following. -->
```

Comment:
Are any of these no longer needed as of latest 16.4 build?

Reply:
Good catch and I think there is mismatch between names.

VC++ expecting Sdks\Microsoft.NET.Sdk\targets\core\Microsoft.NET.Sdk.Core.BeforeCommon.targets

we have Sdks\Microsoft.NET.Sdk\sdk\Sdk.BeforeCommon.targets

I will send email

Reply:
per Olga's email. We cannot remove it yet(blocked on this chance check in and build, and Olga do the change on their side)